client/me/help/help-contact-form/index.jsx

```diff
@@ -77,7 +77,7 @@ export const HelpContactForm = React.createClass( {
 			howYouFeel: 'unspecified',
 			message: '',
 			subject: '',
-			siteSlug: this.getSiteSlug()
+			siteSlug: this.getSiteId()
```

Comment:
Ok... next time I will follow your suggestion; hopefully it won't be anytime soon :)
In this case I've also rebased with master in order to minify the lines in the new pr.

Thanks!